src/main/java/com/amazon/opendistroforelasticsearch/knn/plugin/rest/RestKNNStatsHandler.java

```diff
@@ -73,17 +75,35 @@ public String getName() {
         return NAME;
     }
 
+
+    private List<String> getStatsPath() {
+        List<String> statsPath = new ArrayList<>();
+        statsPath.add("/{nodeId}/stats/");
+        statsPath.add("/{nodeId}/stats/{stat}");
+        statsPath.add("/stats/");
+        statsPath.add("/stats/{stat}");
+        return statsPath;
+    }
+
+    private Map<String, String> getNewURLPathByLegacyURLPath() {
```

Comment:
Nitpicking. This is a bit of a strange name, and capitalizes URL too. This is a legacy to *current* map (there's nothing new about it when this plugin ships), so maybe `getUrlPathToLegacyUrlPathMap()`?

Reply:
Ack